### PR TITLE
Sync docs changes into the embedded help for pwsh

### LINKED ISSUE #9948: pwsh command-line help errors
# Steps to reproduce

```powershell
pwsh -?
```

# Expected behavior

# Actual behavior

There are several issues in the output of help for the command-line options.

1. The `-SettingsFile` parameter is not listed in the syntax block but it is listed and described below.

2. Was `-LoadProfile` actually added? For what purpose? It is not listed in the output for PSv6.

3. The description for `-WorkingDirectory` includes markdown formatting for the "Set-Location" example, which is not appropriate for console output.

```
-WorkingDirectory | -wd
    Sets the working directory at the start of PowerShell given a valid PowerShell directory path.
    Executing `Set-Location -LiteralPath <path>` at startup.
```

4. The the documentation in about_pwsh.md has been improved with more information and better grammar. We should sync those changes into the code.

# Environment data

```none
Name                           Value
----                           -----
PSVersion                      7.0.0-preview.1
PSEdition                      Core
GitCommitId                    7.0.0-preview.1
OS                             Microsoft Windows 10.0.18362
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

### PR DESCRIPTION
**Pull request by sdwheeler**
# PR Summary

Update command-line help for pwsh.exe
Close #9948

## PR Context

The docs for pwsh have been updated with more information and some grammar fixes. This PR syncs those changes back into the exe.
This is related to issue #9948.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [ ] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)

### CODE REVIEW

File: src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx [hunk 1]
```diff
@@ -121,7 +121,7 @@
     <value>PowerShell {0}
 Copyright (c) Microsoft Corporation. All rights reserved.
 
-https://aka.ms/powershell
+https://aka.ms/pscore6-docs
```
Comment (iSazonov) [inline src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx]: It was merged a day before in https://github.com/PowerShell/PowerShell/pull/9775

Comment (sdwheeler) [reply src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx]: Fixed

File: src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx [hunk 2]
```diff
@@ -121,7 +121,7 @@
     <value>PowerShell {0}
 Copyright (c) Microsoft Corporation. All rights reserved.
 
-https://aka.ms/powershell
+https://aka.ms/powershell-docs
```
Comment (SteveL-MSFT) [inline src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx]: We agreed that this should go to the entry page than straight to docs
```suggestion
https://aka.ms/powershell
```

File: src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx [hunk 3]
```diff
@@ -132,145 +132,230 @@ Type 'help' to get help.</value>
...
                   [-Interactive] [-LoadProfile] [-NoExit] [-NoLogo] [-NonInteractive] [-NoProfile]
-                  [-OutputFormat {Text | XML}] [-Version] [-WindowStyle &lt;style&gt;]
-                  [-WorkingDirectory &lt;directoryPath&gt;]
+                  [-OutputFormat {Text | XML}]  [-SettingsFile &lt;filePath&gt;] [-Version]
+                  [-WindowStyle &lt;style&gt;] [-WorkingDirectory &lt;directoryPath&gt;]
 
        pwsh[.exe] -h | -Help | -? | /?
 
-PowerShell Online Help https://aka.ms/powershell-docs
+PowerShell Online Help https://aka.ms/pscore6-docs
```
Comment (SteveL-MSFT) [inline src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx]: This should go to powershell-docs
```suggestion
PowerShell Online Help https://aka.ms/powershell-docs
```

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

**Review by SteveL-MSFT — APPROVED** [review]

File: src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx [hunk 4]
```diff
@@ -144,133 +144,218 @@ All parameters are case-insensitive.</value>
...
+
     Does not exit after running startup commands.
 
-    Example: pwsh -NoExit -Command Get-Date
+    Example: "pwsh -NoExit -Command Get-Date"
 
 -NoLogo | -nol
-    Hides the copyright banner at startup.
+
+Hides the copyright banner at startup.
```
Comment (KalleOlaviNiemitalo) [inline src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx:309]: Filed <https://github.com/PowerShell/PowerShell/issues/13516> about the removal of indentation.

**Pull request merged**